src/tests/java/picard/sam/markduplicates/util/OpticalDuplicateFinderTest.java

```diff
@@ -73,4 +93,26 @@ public void testParseReadName(final String readName, final int tile, final int x
                 {"RUNID:7:1203:2884:16834", 1203, 2884, 16834}
         };
     }
+
+    @Test
+    public void testVeryLongReadNames() {
```

Comment:
I'm going to push back on this ... the default is there really only to show what the code will do.  I think the best answer is to have a symbolic default and call it a day.